qcengine/programs/gaussian.py

```diff
@@ -0,0 +1,269 @@
+
+        if input_model.protocols.native_files == 'all':
+            save_fchk = True
+            gaussian_kw.append('formcheck')
+
+        # Begin input file
+        input_file = []
+        input_file.append('%mem={}MB'.format(int(config.memory * 1024)))
+        input_file.append('#P {}/{}'.format(input_model.model.method, input_model.model.basis) + ' ' + ' '.join(gaussian_kw) + '\n')
+        input_file.append('write your comment here\n')
+  
+        # Create a mol object
+        mol = input_model.molecule
+        input_file.append(f'{int(mol.molecular_charge)} {mol.molecular_multiplicity}')
+
+        # Write the geometry
+        for real, sym, geom in zip(mol.real, mol.symbols, mol.geometry):
+            if real is False:
+                raise InputError('Cannot handle ghost atoms yet.')
+            input_file.append(f'{sym} {geom[0]:14.8f} {geom[1]:14.8f} {geom[2]:14.8f}')
+        input_file.append("\n")
+
+#        print ('*' * 100)
+#        print ('\n'.join(input_file))
+#        print ('*' * 100)
+
+        gaussian_ret = {
+            'infiles': {'input.inp': '\n'.join(input_file)},
+            'commands': [which("g09"),  'input.inp', 'output.log']
+            #'scratch_directory': config.scratch_directory
+            }
+
+        return gaussian_ret
+
+    def execute(self,
+                inputs,
+                *,
+                extra_infiles: Optional[Dict[str, str]] = None,
+                extra_outfiles: Optional[Dict[str, str]] = None,
+                extra_commands: Optional[List[str]] = None,
+                scratch_name = None,
+                timeout: Optional[int] = None
+               ):
+
+        success, dexe = execute(
+            inputs['commands'],
+            inputs['infiles'],
+            outfiles = ['output.log'],
+            scratch_messy = True
+            )
+        
+        if (dexe['outfiles']['output.log'] is None) or (
+           'Error termination via' in dexe['outfiles']['output.log']):
+           print ('THERE IS AN ERROR!')
+           
+           success = False
+        
+        return success, dexe
+ 
+    def parse_output(self, outfiles: Dict[str, str], input_model: AtomicInput) -> AtomicResult:
+        
+        output_data = {}
+        properties = {}
+        cclib_vars = {}
+        
+        tmp_output_path = outfiles['scratch_directory']
+        tmp_output_file = os.path.join(tmp_output_path, 'output.log')
+        data = cclib.io.ccread(tmp_output_file)
+        cclib_vars = data.getattributes(True)
+        
+        last_occupied_energy = data.moenergies[0][data.homos[0]]
+        #output_data['HOMO ENERGY'] = last_occupied_energy
+        
+        scf_energy = data.scfenergies[0] / constants.conversion_factor("hartree", "eV") # Change from the eV unit to the Hartree unit
+        #output_data['SCF ENERGY'] = scf_energy
+        
+        if input_model.driver == 'energy':
+            output_data['return_result'] = scf_energy
```

Comment:
does `data.scfenergies` from cclib store SCF energies or all method energies? If the former, right now you're allowing _any_ `AtIn.model.method` in the input file but setting the `AtRes.return_result` to the SCF value, not the method value.